docs/ref/contrib/gis/functions.txt

```diff
@@ -163,11 +163,20 @@ __ http://www.w3.org/Graphics/SVG/
 
 .. class:: BoundingCircle(expression, num_seg=48, **extra)
 
-*Availability*: `PostGIS <http://postgis.net/docs/ST_MinimumBoundingCircle.html>`__
+*Availability*: `PostGIS <http://postgis.net/docs/ST_MinimumBoundingCircle.html>`__,
+`Oracle (≥ 12.1.0.2) <https://docs.oracle.com/database/121/SPATL/GUID-82A61626-BB64-4793-B53D-A0DBEC91831A.htm#SPATL1554>`_
```

Comment:
I added the link similarly as it done for PostGIS, but not sure if it's good because that link contains Oracle version and probably should be fixed at some point.